### PR TITLE
Alert into dialog

### PR DESCRIPTION
**Pull request by AshokPatel01**

### CODE REVIEW

File: src/app/json-editor/json-editor.component.ts [hunk 1]
```diff
@@ -89,8 +97,8 @@ export class JsonEditorComponent implements OnInit {
           const resSTR = JSON.stringify(json);
           this.jsonSource = JSON.parse(resSTR);
           this.jsonSource = JSON.stringify(JSON.parse(resSTR), null, '    ');
-        } catch (ex) {
-          alert('exception when trying to parse json = ' + ex);
+        } catch (e) {
+          alert('Exception when trying to parse json');
```
Comment (mahavirvora) [inline src/app/json-editor/json-editor.component.ts]: remove javascript alert and show modal dialog

File: src/app/json-editor/json-editor.component.ts [hunk 2]
```diff
@@ -123,7 +131,8 @@ export class JsonEditorComponent implements OnInit {
 
   downloadFile() {
     if (this.jsonTarget.length < 1) {
-      window.alert("This field cant be left empty");
+      // window.alert("This field cant be left empty");
```
Comment (mahavirvora) [inline src/app/json-editor/json-editor.component.ts]: remove unwanted commented code

File: src/app/json-editor/json-editor.component.ts [hunk 3]
```diff
@@ -44,25 +55,22 @@ export class JsonEditorComponent implements OnInit {
...
+      this.handleError();
     }
   }
 
   beautifyJSON() {
     try {
       this.jsonTarget = JSON.stringify(JSON.parse(this.jsonSource), null, '    ');
     } catch (e) {
-      this.handleError(e);
+      this.handleError();
```
Comment (mahavirvora) [inline src/app/json-editor/json-editor.component.ts]: It looks you are not using actual error message, it's showing a static message every time on the modal popup. that should be dynamic

File: src/app/json-editor/json-editor.component.html [hunk 4]
```diff
@@ -12,7 +12,7 @@
             <input type="button" title="Minify" value="Minify" class="btn btn-primary btn-block" (click)="minifyJSON()">
 
             <input type="button" title="JSON Viewer" value="JSON Viewer" class="btn btn-primary btn-block"
-                (click)="jsonViewer()" disabled>
+                (click)="jsonViewer()">
```
Comment (mahavirvora) [inline src/app/json-editor/json-editor.component.html]: Keep jsonViewer button disabled, as it is not working..

Comment (AshokPatel01) [reply src/app/json-editor/json-editor.component.html]: Kept as it is.

File: src/app/json-editor/json-editor.component.ts (hunk 3 above)
Comment (AshokPatel01) [reply src/app/json-editor/json-editor.component.ts]: Resolved it.

File: src/app/json-editor/json-editor.component.ts (hunk 2 above)
Comment (AshokPatel01) [reply src/app/json-editor/json-editor.component.ts]: Removed unwanted commented code.

File: src/app/json-editor/json-editor.component.ts (hunk 1 above)
Comment (AshokPatel01) [reply src/app/json-editor/json-editor.component.ts]: Removed Javascript alert.

File: src/app/json-editor/json-editor.component.ts [hunk 5]
```diff
@@ -15,6 +15,21 @@ export class JsonEditorComponent implements OnInit {
...
   message: any;
   jsonURL: any;
+  @ViewChild('errormessage') errormessage: ElementRef;
+  content: string;
+  title: string;
+
+  errorcontent: string;
+  fileLoadError: string;
+  downloadError: string;
+  loadJsonError: string;
```
Comment (mahavirvora) [inline src/app/json-editor/json-editor.component.ts]: Remove unused variables

Comment (AshokPatel01) [reply src/app/json-editor/json-editor.component.ts]: Removed unused variables

**Review by mahavirvora — APPROVED** [review]

**Pull request merged**